apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -78,6 +78,9 @@ type ClusterResourcePlacementSpec struct {
 	// The number of old ClusterSchedulingPolicySnapshot or ClusterResourceSnapshot resources to retain to allow rollback.
 	// This is a pointer to distinguish between explicit zero and not specified.
 	// Defaults to 10.
+	// +kubebuilder:validation:Minimum=1
+	// +kubebuilder:validation:Maximum=20
```

Comment:
I searched the k8 repo and they assume the max limit is math.MaxInt32 to keep all the revisions.

I cannot think of the reason that customers would like to keep the large number of revisions and 20 should be a reasonable number to start with. It would be easier that we increase the limit if needed compared with adding the limit.

Reply:
20 seems to be a bit low. Some clients may want to keep a long history. why not 1000?